tox.ini

```diff
@@ -29,7 +29,7 @@ deps =
     django30: Django>=3.0,<3.1
     -r{toxinidir}/requirements/test.txt
 commands =
-     celery3: pip install -U celery>=3.1.25,<4.0
+     celery4: pip install -U celery>=3.1.25,<5.0
```

Comment:
I think this line has been unnecessary since August 2017 when the set of environments was updated for Django 1.11 - https://github.com/edx/edx-celeryutils/commit/9542c19994d4521680aabef9725908587c29f67a#diff-b91f3d5bd63fcd17221b267e851608e8